<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deine ToDo's</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineNeueListe">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNeueListe">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Neue Liste</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAktuelleListe">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNeueAufgabe">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>120</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNeueAufgabe">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Neue Aufgabe</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSpeichern"/>
   <addaction name="actionListe_Exportieren"/>
  </widget>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset theme="media-floppy"/>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
   <property name="toolTip">
    <string>Speichern, Strg-S</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionListe_Exportieren">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Liste Exportieren</string>
   </property>
   <property name="toolTip">
    <string>Die ausgew√§hlte Liste als .csv-Datei exportieren</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnNeueListe</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>neueListe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1102</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>997</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNeueAufgabe</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>neueAufgabe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1127</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>984</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineNeueListe</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>neueListe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1139</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>1007</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineNeueAufgabe</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>neueAufgabe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1213</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>1238</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>neueListe()</slot>
  <slot>neueAufgabe()</slot>
 </slots>
</ui>
